app/assets/javascripts/host_edit_interfaces.js

```diff
@@ -388,13 +388,21 @@ $(document).on('change', '.virtual', function() {
     .toggle(!is_virtual);
 });
 
-function update_fqdn() {
-  var host_name = $('#host_name').val();
+function construct_host_name() {
+  var $host_name = $('#host_name')
```

Comment:
Why `$host_name`? is that even valid for a variable name? My JS knowledge is lacking here.

Reply:
I'm marking the jQuery holding variables like that. It is valid, but weird, so maybe better to avoid if it's causing confusion :+1:

Reply:
I agree that it's confusing a bit